lib/checkunusedfunctions.cpp

```diff
@@ -73,7 +73,7 @@ void CheckUnusedFunctions::parseTokens(const Tokenizer &tokenizer, const char Fi
 
         FunctionUsage &usage = _functions[func->name()];
 
-        if (!usage.lineNumber)
+        if (usage.lineNumber == 0)
```

Comment:
this update looks pretty redundant.. I don't care much if we use ! or ==. However ! is a little shorter.

I prefer that you revert these.

Reply:
Reverted